source/index.html.md

```diff
@@ -344,7 +344,7 @@ Provides the ability to send a Payment Request (get paid) to any Contact that ha
 
 To send a Payment Request to a Contact using the API, you must first have an accepted [Agreement](/#Zepto-API-Agreements) with them.
 
-To do so, you can send them an [Open Agreement link](https://help.split.cash/agreements/open-agreement) or [Unassigned Agreement link](http://help.split.cash/agreements/unassigned-agreement) for them to [elect & verify their bank account](https://help.split.cash/bank-accounts/instant-account-verification-iav) and accept the Agreement.
+To do so, you can send them an [Open Agreement link](https://help.split.cash/agreements/open-agreement) or [Unassigned Agreement link](http://help.split.cash/agreements/unassigned-agreement) for them to [elect their bank account](https://help.split.cash/bank-accounts/instant-account-verification-iav) and accept the Agreement.
```

Comment:
I haven't had a chance to look at Agreements in the UI in NZ mode... but I don't think this description is accurate. Also the link points to an article on IAV which is AU based.

Reply:
@chrisixion did you want to have a chat to discuss how to change this? Or should I just remove the link in the meantime?

Reply:
@SmileAndCompile - @Vishakha-Bhosale I believe is going through and changing where the links to the help articles to reflect our product offering in NZ, so this shouldn't be an issue

Reply:
should the link in this description from the config/yml other than hard coded?

Reply:
@henripryde400 the links on the API docs need to be updated to point to the new NZ articles. Happy to send over the new links if that's easy.

Reply:
Yes please @Vishakha-Bhosale . also might make sense to make a new card @henripryde400 ?

Reply:
Just an update - I've updated the agreement links and removed 'elect & verify their bank account'

Reply:
@chrisixion are you OK to approve this now?